scripts/linters/html_linter.py

```diff
@@ -84,37 +83,33 @@ def handle_starttag(self, tag, attrs):
             next_line_column_number = len(next_line) - len(next_line.lstrip())
 
             if next_line_column_number != next_line_expected_indentation:
-                summary_message = (
+                error_message = (
                     '%s --> Expected indentation '
                     'of %s, found indentation of %s '
                     'for content of %s tag on line %s ' % (
                         self.filepath, next_line_expected_indentation,
                         next_line_column_number, tag, line_number + 1))
-                self.summary_messages.append(summary_message)
-                python_utils.PRINT(summary_message)
-                python_utils.PRINT('')
+                self.error_messages.append(error_message)
                 self.failed = True
 
         if tag_line.startswith(opening_tag) and (
                 column_number != expected_indentation):
-            summary_message = (
+            error_message = (
                 '%s --> Expected indentation '
                 'of %s, found indentation of %s '
                 'for %s tag on line %s ' % (
                     self.filepath, expected_indentation,
                     column_number, tag, line_number))
-            self.summary_messages.append(summary_message)
-            python_utils.PRINT(summary_message)
-            python_utils.PRINT('')
+            self.error_messages.append(error_message)
             self.failed = True
 
         if tag not in self.void_elements:
             self.tag_stack.append((tag, line_number, column_number))
             self.indentation_level += 1
 
         if self.debug:
-            python_utils.PRINT('DEBUG MODE: Start tag_stack')
-            python_utils.PRINT(self.tag_stack)
+            concurrent_task_utils.log('DEBUG MODE: Start tag_stack')
```

Comment:
Do we really need this message? [Considering this can come between any different logs and it can be messy!]

Can you please create an issue to check whether this is needed and remove it if not required. [Make sure to add a `TODO(#<issue-id>): ....` message on line 109]

Reply:
Done.